unity-client/Assets/Builder/Scripts/DCLBuilderObjectSelector.cs

```diff
@@ -86,34 +104,36 @@ private void OnMouseDown(int buttonId, Vector3 mousePosition)
             if (buttonId == 0)
```

Comment:
We can invert the if to save indentation.
```suggestion
            if (buttonId != 0)
                return;
```